freeipa/src/main/resources/freeipa-salt/salt/monitoring/scripts/cipa.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+import os
+import subprocess
+import shutil
+import sys
+
+def run_cipa():
+    proc = subprocess.Popen("/usr/local/bin/cipa --output metrics",
+                            stdout=subprocess.PIPE,
+                            stderr=subprocess.PIPE,
+                            stdin=subprocess.PIPE,
+                            shell=True)
+    output, err = proc.communicate()
+    return proc.returncode, output, err
+
+def check_content(fname):
+    with open(fname) as dataf:
+        return any("--output" in line for line in dataf)
+
+def main():
+    main_patched_script="/usr/local/lib/python3.6/site-packages/checkipaconsistency/main.py"
```

Comment:
I'm not sure about python, but can we have this as a constant like thing and all the other constant like variables in the beginning of the script? (I mean all the path to files/folders)

Reply:
yeah, it can, just we need to add global identifier at the start of the script